src/com/books/Book.java

```diff
@@ -0,0 +1,42 @@
+package com.books;
+
+import java.util.Date;
+
+public class Book {
+    private String name;
+
+    public String getName() {
```

Comment:
I think it would be better to place all access-fields methods in the same position at code. Actually Parts of a class always are defined in the respective order: static fields(they should be sorted by access modifier:the most accessible first), instance fields(the same sort rule), constructors and then methods. This rule of element ordering as the other ones for java code you can find in Java Code Convention. It is available at http://www.oracle.com/technetwork/java/codeconventions-150003.pdf . You should follow these rules during writing code using Java.